packages/auth/src/Auth.ts

```
@@ -1075,7 +1075,8 @@ export default class AuthClass {
     public currentSession(): Promise<CognitoUserSession> {
         const that = this;
         logger.debug('Getting current session');
-        if (!this.userPool) { return this.rejectNoUserPool(); }
+        // Purposely not calling the reject method here because we don't need a console error
+        if (!this.userPool) { return Promise.reject(); }
```

This results in silent failure if the user pool was accidentally misconfigured. I don't think that this is expected behaviour - can you clarify why we don't want a console error when the user pool is not configured @iartemiev ?